examples/CIFAR10_with_Eigenvalue_Decay.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import print_function
```

Comment:
I don't think this example is necessary. It only slightly differs from the existing CIFAR10 example.